### PR TITLE
Fix locate module paths

### PR DESCRIPTION
**Pull request by OnlyDeniko**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -204,7 +204,7 @@ def get_build_options():
         ela.append('-s')
     if IS_LIN:
         ela.append("-fPIC")
-        ela.append("-Wl,-rpath,$ORIGIN/../..")
+        ela.append("-Wl,-rpath,$ORIGIN/../../../")
```
Comment (PetrovKP) [inline setup.py:207]: Add too for mac (199 line)

**Comment by OnlyDeniko** [comment]
/intelci: run
set infra_branch=dev/smirnovm-d4p-test-update

**Comment by OnlyDeniko** [comment]
/intelci: run
set infra_branch=dev/dkulandi-d4p-test-update

**Comment by PetrovKP** [comment]
/intelci: restart
set infra_branch=dev/dkulandi-d4p-test-update

**Review by PetrovKP — APPROVED** [review]

**Pull request merged**